### PR TITLE
Release automatization

### LINKED ISSUE #56: Create dev branch and release automatization
As done for eitprocessing. See partially [this PR](https://github.com/EIT-ALIVE/eitprocessing/pull/242).

- [x] Create `develop` branch
- [x] Add action for automatic release
- [x] Register pkg in pypi
- [x] Add instructions for releasing

### PR DESCRIPTION
**Pull request by wbaccinelli**
Added the actions for automatically create a release on GitHub and publishing it on PyPi. The documentation has also been updated.
The actions are mostly taken from `eitprocessing`. 

Closes #56

### CODE REVIEW

**Comment by DaniBodor** [comment]
there is still a [modification](https://github.com/EIT-ALIVE/eitprocessing/pull/263) in process in eitprocessing. Let's wait for that and then fix this one at the same time.

**Comment by DaniBodor** [comment]
@wbaccinelli, the workflow on eitprocessing is now finalized. Can you update this branch with those changes as well, please.

**Comment by wbaccinelli** [comment]
> @wbaccinelli, the workflow on eitprocessing is now finalized. Can you update this branch with [these](https://github.com/EIT-ALIVE/eitprocessing/pull/263/files) changes as well, please.

@DaniBodor done!

File: README.dev.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+Please follow these steps:
+
+1. (**important**) announce your plan to the rest of the community _before you start working_. This announcement should be in the form of a (new) issue;
+1. (**important**) wait until some kind of consensus is reached about your idea being a good idea;
+1. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in changes, possibly from the 'upstream' repository (follow the instructions [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [here](https://help.github.com/articles/syncing-a-fork/));
+1. make sure the existing tests still work by running `pytest` (see also [here](#testing-locally));
+1. add your own tests (if necessary);
+1. update or expand the documentation;
+1. update the `CHANGELOG.md` file with change;
+1. push your feature branch to (your fork of) the eitprocessing repository on GitHub;
```
[the remark this answers is not in the document]
Comment (DaniBodor) [reply README.dev.md:20]: @wbaccinelli , just this update left

**Review by DaniBodor — APPROVED** [review]
thanks for integrating this for the dashboard as well.

one small change was missed, see my comment :)

**Pull request merged**